docs/docs/components/reanimated-drawer-layout.mdx

```diff
@@ -0,0 +1,192 @@
+---
+id: reanimated-drawer-layout
+title: Reanimated Drawer Layout
+sidebar_label: Reanimated Drawer Layout
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Cross-platform replacement for the React Native's [DrawerLayoutAndroid](http://reactnative.dev/docs/drawerlayoutandroid.html) component.  
+For detailed usage of standard parameters, please refer to the [React Native docs](http://reactnative.dev/docs/drawerlayoutandroid.html).
+
+### Usage:
+
+To use it, import it in the following way:
+
+```js
+import ReanimatedDrawerLayout from 'react-native-gesture-handler/ReanimatedDrawerLayout';
+```
+
+## Properties:
+
+### `drawerType`
+
+accepts values of the `DrawerPosition` enum. Defaults to `FRONT`.  
+Specifies the way the drawer will be displayed.
+
+- `FRONT` the drawer will be displayed above the content view.
+- `BACK` the drawer will be displayed below the content view, revealed by sliding away the content view.
+- `SLIDE` the drawer will appear attached to the content view, opening it slides both the drawer and the content view.
+
+| `FRONT`                                               | `BACK`                                               | `SLIDE`                                               |
+| ----------------------------------------------------- | ---------------------------------------------------- | ----------------------------------------------------- |
+| <img src={useBaseUrl('gifs/new-drawer-front.gif')} /> | <img src={useBaseUrl('gifs/new-drawer-back.gif')} /> | <img src={useBaseUrl('gifs/new-drawer-slide.gif')} /> |
+
+### `edgeWidth`
+
+width of the invisible draggable area on the edge of the content view, which can be dragged to open the drawer.
+
+### `hideStatusBar`
+
+a boolean value. When set to `true`, drawer component will use [StatusBar API](http://reactnative.dev/docs/statusbar.html) to hide the OS status bar when the drawer is dragged or idle in the `open` position.
+
+### `statusBarAnimation`
+
+a string, with possible values: `slide`, `none` or `fade`. Defaults to `slide`.
+May be used in combination with `hideStatusBar` to select the animation used for hiding the status bar.
+See [StatusBar API](http://reactnative.dev/docs/statusbar.html#statusbaranimation) docs.
+
+### `overlayColor`
+
+color of the background overlay on top of the content window when the drawer is `open`.  
+This color's opacity animates from 0% to 100% as the drawer transitions from closed to open. Defaults to `rgba(0, 0, 0, 0.7)`.
+
+### `renderNavigationView`
+
+a renderer function for the drawer component.
+Provided with a `progress` SharedValue parameter that indicates the progress of drawer opening/closing. Equals `0` when drawer is `closed` and `1` when `open`.
+This can be used by the drawer component to animated its children while the drawer is opening or closing.
```

Comment:
First of all, I'd probably merge first two sentences as the second one sounds a bit odd on its own. Beside that 

```
Equals `0` when drawer is `closed` and `1` when `open`.
```

What equals `0`? A function? And let's stay consistent, if we use `closed`, then we should also use `opened`.

Either way, I believe it can be rewritten is a better way 😅

Reply:
I agree, does something like this work better?

>a renderer function for the drawer component, provided with a `progress` SharedValue parameter that indicates the progress of drawer opening/closing animation.
>- `progress` equals `0` when the drawer is `closed` and `1` when the drawer is `opened`.
>- `progress` can be used by the drawer component to animated its children while the drawer is opening or closing.

Inspired by the `progress` description used in the `reanimated_swipeable` docs.

Reply:
It is definitely better 😅 What about:

<hr />

a renderer function for the drawer component, provided with a `progress` parameter.

-  `progress` - `SharedValue` that indicates the progress of drawer opening/closing animation.
    - equals `0` when the `drawer` is closed and `1` when the `drawer` is opened
    - can be used by the `drawer` component to animated its children while the `drawer` is opening or closing

Reply:
I agree, changed in [ca3eda4](https://github.com/software-mansion/react-native-gesture-handler/pull/3188/commits/ca3eda4375625b8578bac3fc09e45fdc36af4977)

<details>
<summary>
Collapsed comparison
</summary>

before:
![image](https://github.com/user-attachments/assets/e31ed3e8-73a2-4fae-ad9e-35dede1cf252)
after:
![image](https://github.com/user-attachments/assets/af6d4f9e-6a5e-4b1b-9f37-87458611e693)
</details>